### PR TITLE
(Data Success) Updated Documentation, remove deprecated fields, added missing message fields

### PR DESCRIPTION
**Pull request by admiral-akk**
This PR addresses three issues:
- The documentation is more detailed as to the changes in the last version.
- We remove `SendMessageFieldSet` and `SubmitReviewFieldSet`
- We add `root_content_id` and `subject` to `Message.java`

### CODE REVIEW

File: CHANGES.MD [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+
+- Adds support for Sift Science API Version 205, including new [`$create_content`](https://siftscience.com/developers/docs/curl/events-api/reserved-events/create-content) and [`$update_content`](https://siftscience.com/developers/docs/curl/events-api/reserved-events/update-content) formats
+- V205 APIs are now called -- **this is an incompatible change**
+- Adds support for content decisions to [Decisions API](https://siftscience.com/developers/docs/curl/decisions-api)
+
+
+INCOMPATIBLE CHANGES INTRODUCED IN API V205:
+- `$create_content` and `$update_content` have significantly changed, and the old format will be rejected
+- `$send_message` and `$submit_review` events are no longer valid
+- V205 improves server-side event data validation.  In V204 and earlier, server-side validation accepts some events that did not conform to the published APIs in our [developer documentation](https://siftscience.com/developers/docs/curl/events-api).  V205 does not modify existing event APIs, but may reject invalid event data that were previously accepted.  **Please test your integration on V205 in sandbox before using in production.**
```
Comment (nickhurlburt) [inline CHANGES.MD]: Let's make some small changes to this line for all clients:
`V205 improves server-side event data validation.  In V204 and earlier, server-side validation accepted some events that did not conform to the published APIs in our [developer documentation](https://siftscience.com/developers/docs/curl/events-api).  V205 does not modify existing event APIs other than those mentioned above, but may reject invalid event data that were previously accepted.  **Please test your integration on V205 in sandbox before using in production.**`

I changed "accepts" to "accepted" (to match tense) and added " other than those mentioned above"

**Comment by garylee1** [comment]
Java changes LGTM

File: CHANGES.MD [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+=================
+
+- Adds support for Sift Science API Version 205, including new [`$create_content`](https://siftscience.com/developers/docs/curl/events-api/reserved-events/create-content) and [`$update_content`](https://siftscience.com/developers/docs/curl/events-api/reserved-events/update-content) formats
+- V205 APIs are now called -- **this is an incompatible change**
+- Adds support for content decisions to [Decisions API](https://siftscience.com/developers/docs/curl/decisions-api)
+
+
+INCOMPATIBLE CHANGES INTRODUCED IN API V205:
+- `$create_content` and `$update_content` have significantly changed, and the old format will be rejected
+- `$send_message` and `$submit_review` events are no longer valid
```
Comment (garylee1) [inline CHANGES.MD]: was this ever an event? or just a complex subtype?

**Comment by garylee1** [comment]
actually i have a docs question as well ^

**Comment by admiral-akk** [comment]
Answered the doc question offline.

**Pull request merged**